app/main.development.js

```diff
@@ -179,6 +181,14 @@ const launchDCRWallet = () => {
     args.push('--testnet');
   }
 
+  // Add any extra args if defined.
+  if (argv.extrawalletargs != undefined) {
+    var extraArgs = argv.extrawalletargs.split(' ');
```

Comment:
That probably would break commands that take args with spaces in them but I can live with that.